src/OrbitGl/Batcher.cpp

```diff
@@ -220,15 +220,43 @@ void Batcher::AddTriangle(const Triangle& triangle, const Color& color,
 
 void Batcher::AddTriangle(const Triangle& triangle, const Color& color, const Color& picking_color,
                           std::unique_ptr<PickingUserData> user_data) {
+  std::array<Color, 3> colors;
+  colors.fill(color);
+  AddTriangle(triangle, colors, picking_color, std::move(user_data));
+}
+
```

Comment:
// Draw a shaded trapezium with two sides parallel to the x-axis or y-axis.

Reply:
Done.